pkgs/development/ocaml-modules/curly/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildDunePackage, fetchFromGitHub
+, result, alcotest, cohttp-lwt-unix, odoc, curl }:
+
+buildDunePackage rec {
+  pname = "curly";
+  version = "unstable-2019-11-14";
+
+  minimumOCamlVersion = "4.02";
+
+  src = fetchFromGitHub {
+    owner  = "rgrinberg";
+    repo   = pname;
+    rev    = "33a538c89ef8279d4591454a7f699a4183dde5d0";
+    sha256 = "10pxbvf5xrsajaxrccxh2lqhgp3yaf61z9w03rvb2mq44nc2dggz";
+  };
+
+  propagatedBuildInputs = [ result ];
+  checkInputs = [ alcotest cohttp-lwt-unix ];
+  doCheck = true;
```

Comment:
You should disable tests for OCaml < 4.05, otherwise the `minimumOCamlVersion` attribute has a strange value.

Reply:
good suggestion, thanks, fixed